20150228-wordpress-guids.md

```diff
@@ -0,0 +1,53 @@
+# Post GUIDs: Sometimes You Should Update Them
+
+***TL;DR** &mdash; Updating the GUIDs of a WordPress site that's already live will annoy people subscribed to its RSS feeds. But if the site has never been live, you should update the GUIDs.*
+
+#### What is a GUID?
+
+GUID is an acronym for Globally Unique Identifier. It is a concept that goes well beyond WordPress and is used in all kinds of applications. In the past, I used GUIDs heavily working with ASP.NET and Microsoft SQL Server for example. Those GUIDs looked like this:
+
+```
+6F9619FF-8B86-D011-B42D-00C04FC964FF
+```
+
+(I know, shocking right! A Microsoft stack, really dude? If it's any consolation, I hated every minute of it.)
+
+RSS feeds are another example that employ GUIDs. Every item in an RSS feed can have a `<guid>` element.
+
+#### How does WordPress use GUIDs?
+
+WordPress employs GUIDs when it generates its RSS (and Atom) feeds. If you take a look at the [RSS feed of this blog](https://deliciousbrains.com/blog/feed/) you will see elements similar to this:
+
+```
+<guid isPermaLink="false">https://deliciousbrains.com/?p=6806</guid>
+```
+
+WordPress uses a real, working post URL as the GUID. In fact, if you go to the URL in the example above, it will redirect you to the blog post it belong to. This is also the URL the post has when using the default Permalink setting. Since URLs are globally unique by their nature, they make pretty good GUIDs so long as they don't change. And in this case, since we're using the post ID in the URL, it's almost certain not to change. A good design decision.
+
+#### What are GUIDs used for in RSS feeds?
+
+RSS feed readers (like Feedly) use GUIDs to uniquely identify an item against all other items that have appeared in the feed and will appear in the feed in the future. It uses the GUID to determine if it should show an item as new.
```

Comment:
the feed and will appear **in the feed** in the future

Is this necessary? Feed overload.